GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -1194,6 +1194,15 @@ public void SwitchFocus(bool alreadyContainedFocus)
             }
         }
 
+        public override bool ProcessHotkey(Keys keyData)
+        {
+            return base.ProcessHotkey(keyData) // generic handling of this controls's hotkeys (upstream)
+                || (!GitExtensionsControl.IsTextEditKey(keyData) // downstream (without keys for quick search and filter)
+                    && ((DiffFiles.Visible && DiffFiles.ProcessHotkey(keyData))
+                        || (DiffText.Visible && DiffText.ProcessHotkey(keyData))
+                        || (BlameControl.Visible && BlameControl.ProcessHotkey(keyData))));
```

Comment:
Similar is needed for FileTree subcontrols

Reply:
Added for RevisionFileTreeControl.